src/perl.c

```diff
@@ -32,23 +32,11 @@
 /* do not automatically get the thread specific Perl interpreter */
 #define PERL_NO_GET_CONTEXT
 
-#define DONT_POISON_SPRINTF_YET 1
-#include "collectd.h"
-
-#undef DONT_POISON_SPRINTF_YET
-
-#if HAVE_STDBOOL_H
-# include <stdbool.h>
-#endif
+#include <stdbool.h>
```

Comment:
This seems like an accidental change (merge conflict)... The 5.5.2 branch needs this `#if`/`#endif`.

Reply:
[Optional] The original code had a space here. Might as well keep it consistent with the rest of the code.